README.md

```diff
@@ -6,6 +6,17 @@ The Coolest Platform for the coolest of events, Coolest Projects
 
 To setup the dev environment run `docker-compose run --rm platform yarn`
 
+### Migrations
+
+Migrations are running automatically. In case you want to rerun a migration, knex provides a cli that you can use :
+
+`node ./node_modules/knex/bin/cli.js --knexfile ./server/config/db.json --cwd ./server/database migrate:latest`
```

Comment:
2 things
 1. this work when the db isnt running. eg outside docker 
 1.  would `yarn run knex  --knexfile ./server/config/db.json --cwd ./server/database migrate:latest` not be better, or possible an npm script